app/controllers/admin/payout_reports_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Admin::PayoutReportsController < AdminController
+  attr_reader :payout_report
+
+  def index
+    @payout_reports = PayoutReport.final.paginate(page: params[:page])
+  end
+
+  def show
+    @payout_report = PayoutReport.find(params[:id])
+  end
+
+  def download    
+    require 'tempfile'
+    @payout_report = PayoutReport.find(params[:id])
+
+    send_data payout_report.contents,
+      filename: payout_report_file_name(payout_report),
+      type: :json
+  end
+
+  private
+
+  def payout_report_file_name(payout_report)
+    created_at_str = payout_report.created_at.to_s # "2018-08-17 07:45:23 -0700"
```

Comment:
What if we refactored this to use [`strftime`](https://ruby-doc.org/core-2.2.0/Time.html#method-i-strftime)? I think this might be a little bit less code.

Something like?
`payout_report.created_at.strftime("%FT%H_%M_%S")`

Reply:
@CoryMcDonald Good idea, will update 👍